Redesign The Homepage UI

anupamasok opened the pull request:
Fixes issue #5 

1. Change UI Landing Design
2. Add Favicon
3. Add local Style.css from Tailwindcss classes instead of CDN

Remaining:

1. Add images for each of the page links under **Corona Literacy Mission**, **Corona Care Center**, **CoronaSafe Resources** and **Other Resources**
2. Add Tamil Language Link (in Tamil Characters)
3. Under the Mobile view, Fix layout of Langauge options in the Landing.

**Attached Images After UI Changes**
![image](https://user-images.githubusercontent.com/22481267/77235299-94b2a680-6bda-11ea-9ba4-d0dfe10ef468.png)
![image](https://user-images.githubusercontent.com/22481267/77235282-7b115f00-6bda-11ea-8ac7-d822ef9bd01a.png)

anupamasok committed:
Redesign The Homepage UI

Fixes issue #5

anupamasok committed:
Remove Tailwind.css

bodhish reviewed (changes requested):
Can you re-check the files.
- [ ] `dist/style.css` & `tailwind.css` Do we really need those ?
If we are planing to avoid cdn, It would be better to configure tailwind via parcel so that we dont have to keep this css file on our repo.

anupamasok committed:
Remove Style.css

anupamasok commented:
> Can you re-check the files.
> 
> * [ ]  `dist/style.css` & `tailwind.css` Do we really need those ?
>   If we are planing to avoid cdn, It would be better to configure tailwind via parcel so that we dont have to keep this css file on our repo.

@bodhish Fixed this issue. Reverted to CDN. Removed `dist/style.css` and `tailwind.css`

anupamasok reviewed (commented):
Fixed Issue
